src/CMake/VisItMacros.cmake

```diff
@@ -120,3 +120,196 @@ MACRO(VISIT_PLUGIN_TARGET_RTOD type)
 ENDMACRO(VISIT_PLUGIN_TARGET_RTOD)
 
 
+##############################################################################
+# Function that creates XML tools C++ Code Gen Target 
+##############################################################################
+FUNCTION(ADD_CPP_GEN_TARGET gen_name
+                            src_dir
+                            dest_dir)
+
+    set(gen_target_name "gen_cpp_${gen_name}")
+
+    if(VISIT_CMAKE_VERBOSE_GEN_TARGET_MESSAGES)
+        MESSAGE(STATUS "Adding xml tools C++ generation target: ${gen_target_name}")
+    endif()
+
+    set(xml_gen_tool ${CMAKE_BINARY_DIR}/bin/xml2atts)
```

Comment:
In order to get this working on Windows, I needed to change this line to:
<pre>   
     if(NOT WIN32)
        set(xml_gen_tool ${CMAKE_BINARY_DIR}/bin/xml2atts)
    else()
        set(xml_gen_tool ${VISIT_EXECUTABLE_DIR}/visit.exe -xml2atts)
    endif()
</pre>
Will need to do similar for the other macros.
But in the short term, I think all of this needs to be turned off on windows because the code-gen tools generate files with windows-style line endings, which marks all the files as changed. I am entering a ticket to see if the tools can be changed to output unix-style line endings always.